src/components/cardbuilder/cardBuilder.js

```diff
@@ -1565,12 +1565,24 @@ define(['datetime', 'imageLoader', 'connectionManager', 'itemHelper', 'focusMana
             if (collectionType === 'livetv') {
                 return '<i class="cardImageIcon md-icon">&#xE1B2;</i>';
             }
+            if (collectionType === 'movies') {
+                return '<i class="cardImageIcon md-icon">&#xe04a;</i>';
+            }
+            if (collectionType === 'tvshows') {
+                return '<i class="cardImageIcon md-icon">&#xe333;</i>';
+            }
             if (collectionType === 'homevideos' || collectionType === 'photos') {
-                return '<i class="cardImageIcon md-icon">&#xE412;</i>';
+                return '<i class="cardImageIcon md-icon">&#xe071;</i>';
```

Comment:
I'm not the biggest fan of choosing this icon. I would go for `e413` `photo library` 
![image](https://user-images.githubusercontent.com/3250155/57897491-f8ce4680-7822-11e9-9d63-e226c66add0f.png). It would match with `src/scripts/librarymenu.js`. 

Tho, it might clash with `homevideos`'s icon since both seems to always be linked... If both are always linked, I would choose `photo library` since IMHO this library type is more representative of pictures.

But honestly, I don't have strong opinions here. So both are good with me.

Reply:
Thanks as u suggested i updated homevideos and photo